sdk/core/azure-core/azure/core/pipeline/policies/_retry.py

```diff
@@ -45,6 +44,7 @@
 )
 
 from ._base import HTTPPolicy, RequestHistory
+from ._utils import get_retry_after as _get_retry_after, _parse_retry_after
```

Comment:
maybe just import `_utils` then `_utils.get_retry_after` ? I try to avoid the renames on import